classes/genre.rb

```diff
@@ -0,0 +1,15 @@
+class Genre

```

Comment:
- Optional
Tests for the Genre class have not been implemented. Kindly implement these tests. Since previous reviewer didn't mark this as required, I am marking this as optional.